src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -77,48 +77,52 @@ public class ScriptTest {
     static final String pubkeyProg = "76a91433e81a941e64cda12c6a299ed322ddbdd03f8d0e88ac";
 
-    private static final NetworkParameters PARAMS = TestNet3Params.get();
+    private static final NetworkParameters mainnetParams = MainNetParams.get();
     private static final Logger log = LoggerFactory.getLogger(ScriptTest.class);
-    private final List<BtcECKey> btcECKeyList = new ArrayList<>();
-    private final List<BtcECKey> erpFedECKeyList = new ArrayList<>();
-    private final BtcECKey ecKey1 = BtcECKey.fromPrivate(BigInteger.valueOf(100));
-    private final BtcECKey ecKey2 = BtcECKey.fromPrivate(BigInteger.valueOf(200));
-    private final BtcECKey ecKey3 = BtcECKey.fromPrivate(BigInteger.valueOf(300));
-    private final BtcECKey ecKey4 = BtcECKey.fromPrivate(BigInteger.valueOf(400));
-    private final BtcECKey ecKey5 = BtcECKey.fromPrivate(BigInteger.valueOf(500));
-    private final BtcECKey ecKey6 = BtcECKey.fromPrivate(BigInteger.valueOf(600));
-    private final BtcECKey ecKey7 = BtcECKey.fromPrivate(BigInteger.valueOf(700));
-    private final BtcECKey ecKey8 = BtcECKey.fromPrivate(BigInteger.valueOf(800));
+    private final List<BtcECKey> federationKeys = new ArrayList<>();
+    private final List<BtcECKey> erpFederationKeys = new ArrayList<>();
+    private final BtcECKey fedKey1 = BtcECKey.fromPrivate(BigInteger.valueOf(100));
+    private final BtcECKey fedKey2 = BtcECKey.fromPrivate(BigInteger.valueOf(200));
+    private final BtcECKey fedKey3 = BtcECKey.fromPrivate(BigInteger.valueOf(300));
+    private final BtcECKey fedKey4 = BtcECKey.fromPrivate(BigInteger.valueOf(400));
+    private final BtcECKey fedKey5 = BtcECKey.fromPrivate(BigInteger.valueOf(500));
+    private final BtcECKey fedKey6 = BtcECKey.fromPrivate(BigInteger.valueOf(600));
+    private final BtcECKey fedKey7 = BtcECKey.fromPrivate(BigInteger.valueOf(700));
+    private final BtcECKey fedKey8 = BtcECKey.fromPrivate(BigInteger.valueOf(800));
+    private final Address userAddress = BtcECKey.fromPrivate(BigInteger.valueOf(900)).toAddress(
+        mainnetParams);
 
     @Before
     public void setUp() {
-        btcECKeyList.add(ecKey1);
-        btcECKeyList.add(ecKey2);
-        btcECKeyList.add(ecKey3);
-        erpFedECKeyList.add(ecKey4);
-        erpFedECKeyList.add(ecKey5);
-        erpFedECKeyList.add(ecKey6);
-        erpFedECKeyList.add(ecKey7);
-        erpFedECKeyList.add(ecKey8);
+        federationKeys.add(fedKey1);
+        federationKeys.add(fedKey2);
+        federationKeys.add(fedKey3);
+        erpFederationKeys.add(fedKey4);
+        erpFederationKeys.add(fedKey5);
+        erpFederationKeys.add(fedKey6);
+        erpFederationKeys.add(fedKey7);
+        erpFederationKeys.add(fedKey8);
     }
 
     @Test
     public void testScriptSig() {
+        NetworkParameters testnetParams = TestNet3Params.get();
```

Comment:
why go back to testnet?

Reply:
We could do a separate PR including the required refactor to use mainnet in these tests

Reply:
Task created.